### PR TITLE
Use native conda

### PR DESCRIPTION
**Pull request by proppy**
This makes the notebooks more portable across different environments:
- colab notebooks
- jupyter lab ran from an existing conda environment
- jupyter lab ran outside of a conda environment

Changes:
- use miniconda instead of condacolab: conda-incubator/condacolab#23
- get rid of kernel restart
- skip conda bootstrap if CONDA_PREFIX is defined
- now notebooks can run unmodified w/ jupyter

The diff is not ideal, because of https://github.com/chipsalliance/silicon-notebooks/issues/3

/cc @joamatab

### CODE REVIEW

File: digital-inverter-openlane.ipynb [hunk 1]
```diff
@@ -5,7 +5,8 @@
     "colab": {
```
[the remark this answers is not in the document]
Comment (proppy) [reply digital-inverter-openlane.ipynb]: Done

File: analog-inverter-magic.ipynb [hunk 2]
```diff
@@ -72,25 +43,32 @@
...
-        "  !pip install -q condacolab\n",
-        "  import condacolab\n",
-        "  condacolab.install_from_url('https://repo.anaconda.com/miniconda/Miniconda3-py37_4.11.0-Linux-x86_64.sh')\n",
-        "  !sed -i -e /cudatoolkit/d /usr/local/conda-meta/pinned\n",
-        "  %env CONDA_ROOT=/usr/local"
+        "import os\n",
+        "import pathlib\n",
+        "import sys\n",
+        "\n",
+        "CONDA_PREFIX = os.environ.get('CONDA_PREFIX', None)\n",
```
[the remark this answers is not in the document]
Comment (proppy) [reply analog-inverter-magic.ipynb:52]: Doublec checked that this is defined when the environment is activated.

File: digital-inverter-openlane.ipynb [hunk 3]
```diff
@@ -72,25 +45,33 @@
...
-        "  import condacolab\n",
-        "  condacolab.install_from_url('https://repo.anaconda.com/miniconda/Miniconda3-py37_4.11.0-Linux-x86_64.sh')\n",
-        "  !sed -i -e /cudatoolkit/d /usr/local/conda-meta/pinned\n",
-        "  %env CONDA_ROOT=/usr/local"
+        "import os\n",
+        "import pathlib\n",
+        "import sys\n",
+        "\n",
+        "CONDA_PREFIX = os.environ.get('CONDA_PREFIX', None)\n",
+        "print(CONDA_PREFIX)\n",
```
Comment (proppy) [inline digital-inverter-openlane.ipynb:53]: remove

File: analog-inverter-magic.ipynb [hunk 4]
```diff
@@ -23,11 +7,14 @@
         "colab_type": "text"
       },
       "source": [
-        "<a href=\"https://colab.research.google.com/github/chipsalliance/silicon-notebooks/blob/main/analog-inverter-magic.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+        "<a href=\"https://colab.research.google.com/github/chipsalliance/silicon-notebooks/blob/native-conda/analog-inverter-magic.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
```
Comment (mithro) [inline analog-inverter-magic.ipynb:10]: This should probably stay the same, right?

File: digital-inverter-openlane.ipynb [hunk 5]
```diff
@@ -23,11 +7,14 @@
         "colab_type": "text"
       },
       "source": [
-        "<a href=\"https://colab.research.google.com/github/chipsalliance/silicon-notebooks/blob/main/digital-inverter-openlane.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+        "<a href=\"https://colab.research.google.com/github/chipsalliance/silicon-notebooks/blob/native-conda/digital-inverter-openlane.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
```
Comment (mithro) [inline digital-inverter-openlane.ipynb:10]: Should this stay the same?

**Review by mithro — APPROVED** [review]
LGTM.

File: analog-inverter-magic.ipynb (hunk 4 above)
Comment (proppy) [reply analog-inverter-magic.ipynb:10]: ah, yes nice catch!

File: digital-inverter-openlane.ipynb (hunk 5 above)
Comment (proppy) [reply digital-inverter-openlane.ipynb:10]: ah, yes nice catch!

**Pull request merged**